fitbenchmarking/fitting/controllers/scipy_controller.py

```diff
@@ -33,7 +33,7 @@ def fit(self):
         """
         popt = None
 
-        popt, _ = curve_fit(f=self.functions[self.function_id][0].__call__,
+        popt, _ = curve_fit(f=self.problem.function,
```

Comment:
Should this not be `problem.eval_f`?

Reply:
This doesn't work with eval_f, but I'll add a comment